websites/api-docs-smoke-test/src/content/layouts/endpoint-large-desktop.mdx

```diff
@@ -19,6 +19,11 @@ Lorem ipsum dolor sit amet, everti feugiat gloriatur sed et, an omnes saepe inte
 
 Id quo virtute epicurei phaedrum. No volumus atomorum sit, sonet sensibus ne quo. Tale porro cu qui, ridens option praesent ius ei, nam brute iracundia cu. Aeque soluta necessitatibus te quo, cu vel aperiri euismod delectus. Eu quod invenire mnesarchum sed, his ne decore mollis audire, sit ea aeque zril doctus.
 
+## Test with URL Linking
+
+The [following link](ctp:test:endpoint:/{projectKey}/resource/{id}:POST) should point to the `resource` API endpoint.
```

Comment:
@nkuehn I tried adding endpoint and type link in standard mdx content but it doesn't seem to be supported. 
I'm not sure about how hard it could be adding support for type and endpoint links in standard mdx content. Shall we extend the scope of this ticket covering the use case you pointed out or would you rather add a separate ticket for that?

Reply:
good catch,  I think we simply never completed the feature when it was first implemented.  
The "magic" link handling that makes a lot of the content writing "just work" is here:  https://github.com/commercetools/commercetools-docs-kit/blob/main/packages/gatsby-theme-docs/src/components/link.js

What's blocking is the dependency structure between api docs theme and the general theme. The link resolution logic is pretty monolithic, hard to override / extend by an optional plugin.  

I suggest you leave it out here and open a tech maintenance issue to consider merging the two plugins.  It's in line with the general strategy to reduce the complexity.  The original intent was to not bloat the bundle size of every docs page with all the functionality that is not really needed but since that was decided gatsby's abilities to manually control bundling has gotten much better and we should be able to have just one theme with all features and still make sure the big pieces are loaded separately.

Reply:
Thank you. I've created a placeholder ticket https://github.com/commercetools/commercetools-docs-kit/issues/1450 linking to this thread